### PR TITLE
Add city image component

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: app/MMB/src/mmbList/ImageBadges.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        style={styles.bookingIdBadge}
+        textStyle={styles.badgeText}
+      />
+    </React.Fragment>
+  );
+}
+
+const styles = StyleSheet.create({
+  confirmBadge: {
+    backgroundColor: 'rgba(51, 189, 27, 0.5)',
```
Comment (mrtnzlml) [inline app/MMB/src/mmbList/ImageBadges.js]: Badges should be partially transparent (with blur but we can skip the blur for now)...

File: app/MMB/src/mmbList/CityImage.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  };
+
+  render = () => (
+    <View style={styles.container}>
+      <NetworkImage
+        source={{
+          uri: this.props.imageUrl,
+        }}
+        style={styles.image}
+        resizeMode="stretch"
```
Comment (mrtnzlml) [inline app/MMB/src/mmbList/CityImage.js]: Is this really the best mode? It looks little bit funny on large/small devices. I would go for one with _"scale the image uniformly"_ (see: https://facebook.github.io/react-native/docs/image.html#resizemode).

File: app/shared/src/badge/AdaptableBadge.js [hunk 3]
```diff
@@ -50,7 +62,11 @@ export default function AdaptableBadge(props: Props) {
     <View style={[style.container, props.style]}>
       {props.icon}
       <Text style={[style.text, props.textStyle]}>
-        <Translation passThrough={props.text} />
+        {props.text !== undefined ? (
+          <Translation passThrough={props.text} />
+        ) : (
+          props.translation
```
Comment (mrtnzlml) [inline app/shared/src/badge/AdaptableBadge.js]: Can we just be strict here and accept only translations instead of this? (maybe send separate PR first)

File: app/shared/src/image/StretchedImage.js [hunk 4]
```diff
@@ -14,5 +14,11 @@ const styles = StyleSheet.create({
 });
 
 export default function StretchedImage(props: Object) {
```
Comment (mrtnzlml) [inline app/shared/src/image/StretchedImage.js]: Time to write exact type with allowed values we actually use? 🙂

File: app/MMB/src/mmbList/CityImage.js [hunk 5]
```diff
@@ -0,0 +1,122 @@
+// @flow
```
[the remark this answers is not in the document]
Comment (tbergq) [reply app/MMB/src/mmbList/CityImage.js]: Suggestions?

File: app/MMB/src/mmbList/CityImage.js [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+        </View>
+      </View>
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    height: 152,
+    width: '100%',
```
[the remark this answers is not in the document]
Comment (tbergq) [reply app/MMB/src/mmbList/CityImage.js]: Wouldn't this make it take the size of its parent? so <View style={{width : 400}}>

File: app/MMB/src/mmbList/CityImage.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (mrtnzlml) [reply app/MMB/src/mmbList/CityImage.js]: Just list for example. I don't think it's that important to have MMB twice in the file name/import.

File: app/MMB/src/mmbList/CityImage.js (hunk 6 above)
[the remark this answers is not in the document]
Comment (mrtnzlml) [reply app/MMB/src/mmbList/CityImage.js]: Ok, looks like a good plan... 🙂

File: app/MMB/src/mmbList/CityImage.js (hunk 2 above)
Comment (mrtnzlml) [reply app/MMB/src/mmbList/CityImage.js]: 🙈

File: app/shared/src/badge/AdaptableBadge.js (hunk 3 above)
Comment (tbergq) [reply app/shared/src/badge/AdaptableBadge.js]: Yes, I will create separate PR for this 👍

File: app/MMB/src/mmbList/CityImage.js [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+
+import TravelTypeIcon from './TravelTypeIcon';
+import ImageBadges from './ImageBadges';
+
+type Props = {|
+  imageUrl: string,
+  bookingId: string,
+  type: string,
+  departureCity: string,
+  arrivalCity: string,
```
[the remark this answers is not in the document]
Comment (tbergq) [reply app/MMB/src/mmbList/CityImage.js]: What's the benefit?

[the remark this answers is not in the document]
Comment (mrtnzlml) [reply app/MMB/src/mmbList/CityImage.js]: There is probably none... 😛 OK. Can I check it again?

**Comment by tbergq** [comment]
@mrtnzlml please check again

File: app/MMB/src/mmbList/ImageBadges.js (hunk 1 above)
Comment (mrtnzlml) [reply app/MMB/src/mmbList/ImageBadges.js]: @tbergq What about the opacity? Not now? :)

Comment (mrtnzlml) [reply app/MMB/src/mmbList/ImageBadges.js]: Oh I see - you used opacity but it didn't work? Or it was not so obvious? 🤔

Comment (tbergq) [reply app/MMB/src/mmbList/ImageBadges.js]: it looks different using opacity: 0.5, don't know why 🤷‍♂️

Comment (tbergq) [reply app/MMB/src/mmbList/ImageBadges.js]: > we can skip the blur for now
Also I though you meant to remove it

Comment (mrtnzlml) [reply app/MMB/src/mmbList/ImageBadges.js]: Ok, I'll rephrase: there should be an opacity (I didn't see it) but let's skip the blur effect. Sorry... 🙈 (https://zpl.io/2Gmv4n4)

Comment (tbergq) [reply app/MMB/src/mmbList/ImageBadges.js]: Ok, I will check it on monday

**Comment by mrtnzlml** [comment]
Integrated, thanks.
